nightlybuilds.php

```diff
@@ -113,14 +113,17 @@ public function onContentPrepare($context, &$article, &$params, $page = 0)
 				if ($branch == JVersion::RELEASE)
 				{
 					$linkedBranch = 'staging';
+					$updateserver = 'https://update.joomla.org/core/nightlies/next_patch_list.xml';
 				}
 				elseif ($branch == '3.7')
 				{
 					$linkedBranch = '3.7.x';
+					$updateserver = 'https://update.joomla.org/core/nightlies/next_minor_list.xml';
 				}
 				else
 				{
 					$linkedBranch = "$branch-dev";
+					$updateserver = 'https://update.joomla.org/core/nightlies/next_major_list.xml';
```

Comment:
So the one thing that won't work consistently is how this `$updateserver` variable is defined.  The 3.7 branch broke the "standard" naming convention for branches we'd been using (normally they are `x.y-dev`) which is why it has a special case here.

The other issue is this entire if conditional only runs if the branch `.txt` file(s) aren't transferred as part of the nightly build job.  So for now this actually would cause undefined variable issues on the current site.